### PR TITLE
Rewrites tests to JavaScript

### PR DESCRIPTION
**Pull request by artem-zakharchenko**
## Change log

> This pull request is a step of code base decaffeination. It's not meant to be merged into `master`, and it heading to the feature branch.

- Rewrites tests from CoffeeScript to JavaScript
- Retains test suits as-is
- Updates validation suits when body is not present (asserts it throws), `validatable-http-message-test.js@194`

## GitHub

* Relates to #96

### CODE REVIEW

**Comment by artem-zakharchenko** [comment]
**This branch must be merged after #132 is merged.**

File: src/meta-schema-v3.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+module.exports = {
```
Comment (honzajavorek) [inline src/meta-schema-v3.js]: An idea for a separate issue/PR: I think these meta schemas might make more sense as static .json files

**Comment by honzajavorek** [comment]
Now this needs rebase

```bash
$ git pull --rebase origin 96-js-version
$ git push origin 96-js-tests -f
```

**Comment by artem-zakharchenko** [comment]
Rebased upon `origin/decaffeinate`.

**Comment by honzajavorek** [comment]
I retried the flaky test and will merge this. We can do the Cucumber test steps 🥒 in a next PR.

**Pull request merged**